integration_tests/src/main/python/string_test.py

```diff
@@ -498,16 +498,26 @@ def test_re_replace():
                 'REGEXP_REPLACE(a, "TEST", NULL)'),
         conf=_regexp_conf)
 
-@allow_non_gpu('ProjectExec', 'RegExpReplace')
 def test_re_replace_backrefs():
-    gen = mk_str_gen('.{0,5}TEST[\ud720 A]{0,5}')
-    assert_gpu_fallback_collect(
+    gen = mk_str_gen('.{0,5}TEST[\ud720 A]{0,5}TEST')
```

Comment:
Why is the test modified to remove the `TEST` at the end of the generating string?